tests/slurm_plugin/test_fleet_manager.py

```diff
@@ -30,46 +29,14 @@ def boto3_stubber_path():
 
 class TestFleetManagerFactory:
     @pytest.mark.parametrize(
-        ("config_file_content", "expected_failure", "expected_manager"),
+        ("fleet_config", "expected_failure", "expected_manager"),
         [
-            (
-                None,
-                "Unable to read",
-                None,
-            ),
-            (
-                {"Scheduling": {"SlurmQueues": [{"Name": "bad_queue", "ComputeResources": [{"Name": "cr1"}]}]}},
-                "Unable to found queue .* or compute resource .* in the .*",
-                None,
-            ),
-            (
-                {"Scheduling": {"SlurmQueues": [{"Name": "q1", "ComputeResources": [{"Name": "bad_cr"}]}]}},
-                "Unable to found queue .* or compute resource .* in the .*",
-                None,
-            ),
-            (
-                {"Scheduling": {"SlurmQueues": [{"Name": "q1", "ComputeResources": [{"Name": "cr1"}]}]}},
-                "InstanceTypeList or InstanceType field not found .*",
-                None,
-            ),
-            (
-                {
-                    "Scheduling": {
-                        "SlurmQueues": [{"Name": "q1", "ComputeResources": [{"Name": "cr1", "InstanceType": "x"}]}]
-                    }
-                },
-                None,
-                Ec2RunInstancesManager,
-            ),
-            (
-                {
-                    "Scheduling": {
-                        "SlurmQueues": [{"Name": "q1", "ComputeResources": [{"Name": "cr1", "InstanceTypeList": "x"}]}]
-                    }
-                },
-                None,
-                Ec2CreateFleetManager,
-            ),
+            ({}, "Unable to found queue .* or compute resource .* in the fleet config: {}", None),
+            ({"bad_queue": {}}, "Unable to found queue .* or compute resource .* in the .*", None),
+            ({"q1": {"bad_cr": {}}}, "Unable to found queue .* or compute resource .* in the .*", None),
+            ({"q1": {"cr1": {}}}, "InstanceTypeList or InstanceType field not found .*", None),
+            ({"q1": {"cr1": {"InstanceType": "x"}}}, None, Ec2RunInstancesManager),
+            ({"q1": {"cr1": {"InstanceTypeList": "x"}}}, None, Ec2CreateFleetManager),
```

Comment:
Let's add a test for a queue with both types of compute resources.

Reply:
Added two compute resources in both the tests, thanks!